RecoHGCal/TICL/python/TrkStep_cff.py

```diff
@@ -32,7 +32,7 @@
   algo_verbosity = 2,
   oneTracksterPerTrackSeed = True,
   promoteEmptyRegionToTrackster = True,
-  itername = "TRK"
+  itername = "TrkHAD"
```

Comment:
My feeling is that this PR will not pass the tests. If you couple `itername` with the `inputTag` used in validation, prepended with the string `ticlMultiClustersFromTracksters`, this `itername` will have no corresponding tag.

Reply:
RIght, I propagated the new `itername` to all python labels with the latest commit.